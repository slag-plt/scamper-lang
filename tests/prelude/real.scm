(real? 56)
(real? 56.2)
(real? "56.2")
(real? (/ 50 2))
(real? (/ 51 2))