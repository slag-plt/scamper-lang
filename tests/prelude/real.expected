#f
#t
#f
#f
#t
